### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by Morpheus636** [framewiki/framewiki.net#32] [Announcements]
@wiertel @KellyNyanbinary @BoredFishRE @jyancat @ItsTrashPanda I'm now realizing I forgot to add a license to this repo. Is everyone OK with [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/?ref=chooser-v1)? I selected it because it's what Wikipedia uses.

If you agree, please reply "Yes, I agree to license my Framewiki contributions under CC BY-SA 4.0" in this thread so that I can add the license to the repo. Otherwise, please suggest an alternative license that you would agree to use. Thanks!

For the record: Yes, I agree to license my Framewiki contributions under CC BY-SA 4.0. As soon as I have permission from everyone mentioned above, I'll push the license.

**Comment by BoredFishRE** [comment]
Yes, I agree to license my Framewiki contributions under CC BY-SA 4.0

**Comment by ItsTrashPanda** [comment]
Yes, I agree to license my Framewiki contributions under CC BY-SA 4.0

**Comment by KellyNyanbinary** [comment]
Yes, I agree to license my Framewiki contributions under CC BY-SA 4.0 

PS 
Amused that I've been cloning and editing a repo that was for all intents and purposes all rights reserved. Hopefully no harm done.

**Reply by Morpheus636** [reply]
Lol it's certainly no problem by me! I normally use my template repo which includes a license, but I used someone else's template (which is licensed under the MIT license, it's just not included in the repo) for this repo and forgot to add one.

**Comment by jyancat** [comment]
Yes, I agree to license my Framewiki contributions under CC BY-SA 4.0

**Comment by Morpheus636** [comment]
Heard back from @wiertel via forum DMs so I'm going to push the license now. Thanks, for all your help!
